Classes/EventListener/Order/Create/PersistOrder/Products.php

```diff
@@ -185,15 +184,6 @@ protected function addVariantsOfVariant(BeVariant $variant, int $level): void
      */
     protected function addBeVariant(BeVariant $variant, int $level): void
     {
-        $orderTax = GeneralUtility::makeInstance(
-            Tax::class,
-            $variant->getTax(),
-            $this->taxClasses[$variant->getTaxClass()->getId()]
-        );
-        $orderTax->setPid($this->storagePid);
-
-        $this->taxRepository->add($orderTax);
```

Comment:
Think the `$this->taxRepository` isn't needed anymore in this class. Could the property and inject-method can also be removed?

Reply:
@extcode I have unfortunately no laptop with me, just a smartphone with the GitHub app. If you want to merge it quick you need to adapt the code yourself. All the proposals look good to me.